lib/rdstation/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RDStation
-  VERSION = '2.8.1'
```

Comment:
Não tenho certeza se não estamos causando uma breaking change aqui, através da remoção do campo details, quando o erro é proveniente do RDSM. Assim como o CRM, pode acontecer de outros usuários desse client também ter a dependencia desse campo de detalhes que antes fazia parte da interface pública dessa gema.

O que você acha sobre isso @Hil?

Reply:
I agree, @angeliski what do you think about this? 👀

Reply:
I'll leave some suggestions here:

1. If we are still in transition, where both response types will be returned, I would keep the field, and just add the gateway response, without removing the previous behavior.
2. If we're really going to change all the responses, I recommend adjusting the major, because the response, even with errors, is part of the public interface of this gem.

I suggest we have some Breaking Changes Policy (eg [Linkedin](https://learn.microsoft.com/en-us/linkedin/shared/breaking-change-policy#definition)), but that's another discussion

Reply:
Agree with @masisiliani 

Remove this information is "bad" for our clients. We need to implement a transition flow AND set a deprecation for those fields. When we relase a `3.0.0` we can remove this without worry